admin/sql/updates/schema-change/25.slave.sql

```diff
@@ -9,4 +10,147 @@ SELECT '20180503-mbs-9708-drop-area-containment-view.sql';
 
 DROP VIEW area_containment;
 
+--------------------------------------------------------------------------------
+SELECT '20190423-mbs-10052-eaa.sql';
+
+SET search_path = 'event_art_archive';
+
+CREATE TYPE musicbrainz.event_art_presence AS ENUM ('absent', 'present', 'darkened');
+
+ALTER TABLE musicbrainz.event_meta
+ADD COLUMN event_art_presence event_art_presence NOT NULL DEFAULT 'absent';
+
+-- Tables
+
+CREATE TABLE art_type ( -- replicate (verbose)
+    id                  SERIAL NOT NULL, -- PK
+    name                TEXT NOT NULL,
+    parent              INTEGER, -- references event_art_archive.art_type.id
+    child_order         INTEGER NOT NULL DEFAULT 0,
+    description         TEXT,
+    gid                 uuid NOT NULL
+);
+
+INSERT INTO art_type VALUES
+    (1, 'Poster', NULL, 0, NULL, generate_uuid_v3('6ba7b8119dad11d180b400c04fd430c8', 'even_art_archive.art_type' || 1));
```

Comment:
Let me know if there's other types we want to add. I added poster at least because the index listing query relies on `type_id = 1` for the `is_front` calculation.

Reply:
Most probably “Flyer” (or “Leaflet”) and “Ticket”, but it seems reasonable to ask the community about their needs.

Reply:
Agreed with just letting the community decide on others. Do we need to tell the IA specifically if we add more?

Reply:
I don't think so. They were mainly concerned about how to select a representative image, since they use a type=Front check for the CAA.